src/lightning/app/utilities/app_helpers.py

```diff
@@ -592,7 +592,7 @@ def _handle_is_headless(app: "LightningApp"):
 
     from lightning.app.utilities.network import LightningClient
 
-    client = LightningClient()
+    client = LightningClient(retry=True)
```

Comment:
This doesn't help to solve the problem though. The default value for `retry` is already `True`

Reply:
Yes, this isn't the fix. True is the default value.